### PR TITLE
Add Conditional Rendering to New Assets Page

### PR DESCRIPTION
**Pull request by salmad3**
Addresses: https://github.com/w3f/polkadot-wiki/pull/2718#issuecomment-941470957

### CODE REVIEW

File: docs/learn/learn-assets.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 id: learn-assets
-title: Assets on Polkadot
-sidebar_label: Assets on Polkadot
-description: Learn about fungible and non-fungible assets in the Polkadot network.
+title: Assets
+sidebar_label: Assets
+description: Learn about fungible and non-fungible assets in the Polkadot ecosystem.
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]: This should definitely stay "network"

**Comment by joepetrowski** [comment]
Made some changes to the base branch from `32f4845` but don't agree with all of them. Better to keep this PR highly on point: just the pieces to make conditional rendering. We can address other stuff in the main PR.

File: docs/learn/learn-assets.md (hunk 1 above)
Comment (salmad3) [reply docs/learn/learn-assets.md]: Updated

File: docs/learn/learn-assets.md [hunk 2]
```diff
@@ -1,50 +1,59 @@
...
-Statemint does not support smart contracts. See the [Advanced](#advanced-techniques) section at the
-bottom for discussion on using proxy and multisig accounts to replicate oft used contract logic.
+Assets in the {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} network can be 
+represented on several chains. They can also take many forms, from a parachain's native token 
+to on-chain representations of off-chain reserves. This page focuses on the latter, namely assets 
+that would be issued by a creator (e.g. rights to audited, off-chain reserves held by the creator, 
+or art issued as an NFT).
+
+The {{ polkadot: [Statemint parachain](https://www.parity.io/blog/statemint-generic-assets-chain-proposing-a-common-good-parachain-to-polkadot-governance/) :polkadot }} 
+{{ kusama: Statemine :kusama }} hosts data structures and logic that
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]: Blog post is still relevant.

```suggestion
{{ kusama: [Statemineparachain](https://www.parity.io/blog/statemint-generic-assets-chain-proposing-a-common-good-parachain-to-polkadot-governance/) :kusama }} hosts data structures and logic that
```

File: docs/learn/learn-assets.md [hunk 3]
```diff
@@ -1,50 +1,59 @@
...
+specialize in the creation, management, and use of assets in the {{ polkadot: Polkadot :polkadot }} 
+{{ kusama: Kusama :kusama }} network. Although other
+parachains can host applications dealing with assets on {{ polkadot: Statemint, Statemint :polkadot }} 
+{{ kusama: Statemine, Statemine :kusama }} can be thought of as the "home base" of assets in the 
+network.
+
+{{ polkadot: Statemint uses DOT as its native token :polkadot }} 
+{{ kusama: Statemine uses KSM as its native token :kusama }}. The chain yields its governance to
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]:
```suggestion
{{ polkadot: Statemint uses DOT :polkadot }} 
{{ kusama: Statemine uses KSM :kusama }} as its native token. The chain yields its governance to
```

File: docs/learn/learn-assets.md [hunk 4]
```diff
@@ -1,50 +1,59 @@
...
+its respective Relay Chain, and has no inflation or era-based rewards for collators. Collators
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]:
```suggestion
its parent Relay Chain, and has no inflation or era-based rewards for collators. Collators
```

File: docs/learn/learn-assets.md [hunk 5]
```diff
@@ -1,50 +1,59 @@
...
+receive a portion of transaction fees. As a
+[common good parachain](https://polkadot.network/blog/common-good-parachains-an-introduction-to-governance-allocated-parachain-slots/), {{ polkadot: Statemint 
+has a trusted relationship with the Relay Chain, 
+and as such, can teleport DOT between itself and its Relay Chains. That is, DOT on Statemint is
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]:
```suggestion
and as such, can teleport DOT between itself and the Relay Chain. That is, DOT on Statemint is 
```

File: docs/learn/learn-assets.md [hunk 6]
```diff
@@ -1,50 +1,59 @@
...
+{{ polkadot: Statemint uses DOT as its native token :polkadot }} 
+{{ kusama: Statemine uses KSM as its native token :kusama }}. The chain yields its governance to
+its respective Relay Chain, and has no inflation or era-based rewards for collators. Collators
+receive a portion of transaction fees. As a
+[common good parachain](https://polkadot.network/blog/common-good-parachains-an-introduction-to-governance-allocated-parachain-slots/), {{ polkadot: Statemint 
+has a trusted relationship with the Relay Chain, 
+and as such, can teleport DOT between itself and its Relay Chains. That is, DOT on Statemint is 
+just as good as DOT on the Relay Chain. :polkadot }}
+{{ kusama: Statemine has a trusted relationship with the Relay Chain, 
+and as such, can teleport KSM between itself and its Relay Chains. That is, KSM on Statemine is
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]:
```suggestion
and as such, can teleport KSM between itself and the Relay Chain. That is, KSM on Statemine is 
```

File: docs/learn/learn-assets.md [hunk 7]
```diff
@@ -1,50 +1,59 @@
...
 metered and instead are benchmarked upon every release, leading to efficient execution and stable
 transaction fees.
 
 ### Creation and Management
 
-Anyone on the network can create assets on Statemint, as long as they can reserve the required
-deposit (1 KSM on Statemine, 100 DOT on Statemint). The network reserves the deposit on creation.
+Anyone on the network can create assets on {{ polkadot: Statemint as long as they can reserve the 
+required deposit of 100 DOT on Statemint. :polkadot }} {{ kusama: Statemine as long as they can reserve 
+the required deposit of 1 KSM on Statemine. :kusama }}. The network reserves the deposit on creation.
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]:
```suggestion
Anyone on the network can create assets on {{ polkadot: Statemint, as long as they can reserve the 
required deposit of 100 DOT. :polkadot }} {{ kusama: Statemine, as long as they can reserve 
the required deposit of 1 KSM. :kusama }}. The network reserves the deposit on creation.
```

File: docs/learn/learn-assets.md [hunk 8]
```diff
@@ -55,78 +64,85 @@ privileged roles, but can reassign them after creation. These roles are:
...
 An asset's details contain one field not accessible to its owner or admin team, that of asset
-sufficiency. Only the network's governance mechanism can deem an asset as _sufficient._ A balance of
+sufficiency. Only the network's governance mechanism can deem an asset as *sufficient.* A balance of
 a non-sufficient asset (the default) can only exist on already-existing accounts. That is, a user
 could not create a new account on-chain by transferring an insufficient asset to it; the account
-must already exist by having more than the existential deposit in DOT (or a sufficient asset). However, assets deemed
-_sufficient_ can instantiate accounts. In the future, _sufficient_ assets will be able to pay
-transaction fees, such that users can transact on Statemint without the need for DOT.
+must already exist by having more than the existential deposit {{ polkadot: in DOT :polkadot }} 
+{{ kusama: in KSM :kusama }}. However, assets deemed *sufficient* can instantiate accounts. In the
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]:
```suggestion
must already exist by having more than the existential deposit {{ polkadot: in DOT :polkadot }} 
{{ kusama: in KSM :kusama }} (or a sufficient asset). However, assets deemed *sufficient* can instantiate accounts. In the 
```

File: docs/learn/learn-assets.md [hunk 9]
```diff
@@ -55,78 +64,85 @@ privileged roles, but can reassign them after creation. These roles are:
...
+as long as they reserve the required deposit of 1 KSM on Statemine :kusama }}. Creating instances of a 
+class also requires a per-instance deposit, unless the chain's governance designates the class as 
+"free holding", allowing the class to mint more instances without deposit. The creator must specify a 
+`ClassId`, which, like its cousin `AssetId`, should be the canonical identifier for the class.
 
 The creator can also specify the same privileged roles of Owner, Admin, Issuer, and Freezer.
 
 Asset classes and instances can have associated metadata. The metadata is an array of data that the
-class Owner can add on-chain, for example a link to an IPFS hash or other off-chain hosting service.
+class Owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting services.
```
Comment (joepetrowski) [inline docs/learn/learn-assets.md]: The "other" implies that it's one of many.
```suggestion
class Owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting service.
```

**Review by joepetrowski — COMMENTED** [review]
Just a few nits, then good to go

**Pull request merged**